### PR TITLE
Alessio fix mockup

### LINKED ISSUE #205: Billing and payment custom form and flow mockup
Realization of the payment and billing flow mockup based on the specs contained in the issue #165 . Generation of related design specs

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:
Apply fix review

Closes #205 .

#

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Complexity increasing per file
==============================
- src/stripe/Components/InvoiceForm.js  2
- src/stripe/Components/InfoPriceColumn.js  1
- src/stripe/Components/PriceTable.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218&bid=19582651)

File: assets/css/payment_template.css [hunk 1]
```diff
@@ -334,3 +333,13 @@ html {
 .fas:hover + .tooltip {
 	display: inline-block;
 }
+
+@media screen and (max-width: 1023px) {
+	.one-quarter{
```
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:338]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css [hunk 2]
```diff
@@ -334,3 +333,13 @@ html {
...
+		width: 100%;
+	}
+	.card, .features-list{
```
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/stripeStyle.css [hunk 3]
```diff
@@ -544,7 +549,8 @@ table tr {
 }
 
 .error-message {
-	color: #080808;
+	color: #D32F2F;
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:552]: Codacy found an issue: [Expected "#D32F2F" to be "#d32f2f" (color-hex-case)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:338]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 2 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected newline after "," (selector-list-comma-newline-after)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/stripeStyle.css [hunk 4]
```diff
@@ -573,3 +579,10 @@ table tr {
 .coupon-message {
 	height: 8px;
 }
+.coupon-notice {
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:582]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/stripeStyle.css [hunk 5]
```diff
@@ -573,3 +579,10 @@ table tr {
...
+	color: #ffd600;
+}
+
+.description {
+	font: 400 14px/19px "Open Sans", sans-serif;
+}
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:588]: Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Complexity increasing per file
==============================
- src/stripe/Components/InvoiceForm.js  2
- src/stripe/Components/InfoPriceColumn.js  1
- src/stripe/Components/PriceTable.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218&bid=19582651)

File: assets/css/stripeStyle.css (hunk 3 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:552]: Codacy found an issue: [Expected "#D32F2F" to be "#d32f2f" (color-hex-case)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:338]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 2 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected newline after "," (selector-list-comma-newline-after)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/stripeStyle.css (hunk 4 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:582]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 2 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:341]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/stripeStyle.css (hunk 5 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:588]: Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: assets/css/payment_template.css (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/css/payment_template.css:338]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)

File: src/stripe/Components/InvoiceForm.js [hunk 6]
```diff
@@ -88,13 +91,19 @@ export default function CheckoutForm(props) {
 		destination_code: '0000000',
 	};
 	if (invoicePreValues) {
+		if (invoicePreValues.prefix.charAt(0) !== '+') {
+			invoicePreValues.prefix = '+' + invoicePreValues.prefix;
```
Comment (HyprCoder) [inline src/stripe/Components/InvoiceForm.js]:
```suggestion
			invoicePreValues.prefix = `+${invoicePreValues.prefix}`;
```

**Review by HyprCoder — CHANGES_REQUESTED** [review]

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  3
- src/stripe/Components/InfoPriceColumn.js  1
- src/stripe/Components/InvoiceForm.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218&bid=19582651)

**Review by HyprCoder — APPROVED** [review]

**Pull request merged**